crates/oxc_linter/src/rules/eslint/no_multi_str.rs

```diff
@@ -0,0 +1,74 @@
+use lazy_static::lazy_static;
+use oxc_ast::AstKind;
+use oxc_diagnostics::OxcDiagnostic;
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+use regex::Regex;
+
+use crate::{context::LintContext, rule::Rule, AstNode};
+
+fn no_multi_str_diagnostic(span0: Span) -> OxcDiagnostic {
+    OxcDiagnostic::warn("eslint(no-multi-str): .").with_label(span0)
```

Comment:
Looks like we forget to add error message :)

Reply:
Woops, added locally, will push when I resolved the other comments.